apps/web-giddh/webpack.electron.partial.js

```diff
@@ -24,6 +24,7 @@ module.exports = {
             'LINKEDIN_CLIENT_ID': JSON.stringify(process.env.LINKEDIN_CLIENT_ID_PROD),
             'LINKEDIN_SECRET_KEY': JSON.stringify(process.env.LINKEDIN_SECRET_KEY_PROD),
             'RAZORPAY_KEY': JSON.stringify(process.env.RAZORPAY_KEY_TEST),
+            'NODE_API_URL': JSON.stringify('http://localhost:8080/'),
```

Comment:
Consider using:
```suggestion
            'NODE_API_URL': JSON.stringify(process.env.NODE_API_URL),
```

Reply:
We should also handle the scenarios where the port `8080` is not available so that the server can be started on a different port.